src/Core/src/Platform/Android/IAndroidLifecycleHandler.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Android.App;
+using Android.OS;
+
+namespace Microsoft.Maui
+{
+	/// <summary>
+	/// Allow to get Android Activity lifecycle callbacks.
+	/// </summary>
+	public interface IAndroidLifecycleHandler : IPlatformLifecycleHandler
```

Comment:
Let's not forget to add them all... There's at least a few missing:

 - OnRestart
 - OnSaveInstanceState
 - OnConfigurationChanged
 - OnActivityResult
 - ....

We need to do a full audit of the `Activity` type and add them all.

Reply:
Included more methods to cover all the Activity possibilities.